src/main/java/seedu/notus/data/notebook/Note.java

```diff
@@ -121,6 +121,10 @@ public boolean getIsArchived() {
         return this.isArchived;
     }
 
+    public String getIsArchivedString() {
+        return (isArchived ? "Archived" : "Unarchived");
```

Comment:
I feel that can return "" for unarchived, don't really need to state that a normal note is unarchived